### PR TITLE
BD-1979: Update time saving details

### PR DESCRIPTION
**Pull request by lydia-xie**
# Pull Request/Issue Resolution

Closes #**ISSUE_NUMBER_HERE**

#### Is this change associated with a Braze feature/product release?
- [ ] Yes (**Insert Feature Release Date Here**)
- [ ] No

---

### CODE REVIEW

File: _docs/_user_guide/data_and_analytics/custom_data/custom_attributes/nested_custom_attribute_support.md [hunk 1]
```diff
@@ -129,6 +129,8 @@ To capture dates as object properties, you must use the `$time` key. In the foll
 }
 ```
 
+Note that for nested custom attributes, if the year is less than 0 or greater than 3000, Braze does not these values on the user.
```
Comment (KellieHawks) [inline _docs/_user_guide/data_and_analytics/custom_data/custom_attributes/nested_custom_attribute_support.md:132]: "does not these values on the user.", add?

**Review by KellieHawks — CHANGES_REQUESTED** [review]
Looks great! Left one comment.

File: _docs/_user_guide/data_and_analytics/custom_data/custom_attributes/nested_custom_attribute_support.md (hunk 1 above)
Comment (lydia-xie) [inline _docs/_user_guide/data_and_analytics/custom_data/custom_attributes/nested_custom_attribute_support.md:132]:
```suggestion
Note that for nested custom attributes, if the year is less than 0 or greater than 3000, Braze does not add these values to the user.
```

File: _docs/_api/objects_filters/user_attributes_object.md [hunk 2]
```diff
@@ -81,7 +81,7 @@ The following data types can be stored as a custom attribute:
...
+| Dates | Must be stored in the [ISO 8601][19] format or in any of the following formats: <br>- `yyyy-MM-ddTHH:mm:ss:SSSZ` <br>- `yyyy-MM-ddTHH:mm:ss` <br>- `yyyy-MM-dd HH:mm:ss` <br>- `yyyy-MM-dd` <br>- `MM/dd/yyyy` <br>- `ddd MM dd HH:mm:ss.TZD YYYY` <br><br>Note that "T" is a time designator, not a placeholder, and should not be changed or removed. <br><br>Time attributes without a time zone will default to Midnight UTC (and will be formatted on the dashboard as the equivalent of Midnight UTC in the company's time zone). <br><br> Events with timestamps in the future will default to the current time. <br><br> For regular custom attributes, if the year is less than 0 or greater than 3000, Braze stores these values as strings on the user. |
```
Comment (lydia-xie) [inline _docs/_api/objects_filters/user_attributes_object.md:84]:
```suggestion
| Dates | Must be stored in the [ISO 8601][19] format or in any of the following formats: <br>- `yyyy-MM-ddTHH:mm:ss:SSSZ` <br>- `yyyy-MM-ddTHH:mm:ss` <br>- `yyyy-MM-dd HH:mm:ss` <br>- `yyyy-MM-dd` <br>- `MM/dd/yyyy` <br>- `ddd MM dd HH:mm:ss.TZD YYYY` <br><br>Note that "T" is a time designator, not a placeholder, and should not be changed or removed. <br><br>Time attributes without a time zone will default to Midnight UTC (and will be formatted on the dashboard as the equivalent of Midnight UTC in the company's time zone). <br><br> Events with timestamps in the future will default to the current time. <br><br> For regular custom attributes, if the year is less than 0 or greater than 3000, Braze stores these values as strings on the user. |
```

**Review by KellieHawks — APPROVED** [review]

**Pull request merged**